canonical-web-and-design/ubuntu.com
# Move /core/build to /build

## Done

- Changed `/core/build` to `/build`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/build and make sure it works

## Issue / Card

Fixes [web-squad#2434](https://github.com/canonical-web-and-design/web-squad/issues/2434)
